src/SDL/main.m

```diff
@@ -129,6 +134,30 @@ int main(int argc, char *argv[])
 				if (i < argc)
 					[controller setPlayerFileToLoad: [NSString stringWithCString: argv[i]]];
 			}
+
+   			if (!strcmp("-help", argv[i]) || !strcmp("--help", argv[i]))
+			{
+				char const *processName = [[[NSProcessInfo processInfo] processName] UTF8String];
+				char s[1024];
+				sprintf(s,	"Usage: %s [options]\n\n"
```

Comment:
Should be `snprintf` or, ideally, `asprintf`. (Okay, it's safe unless the process name is unusually long, but as a matter of principle…)